### DISCUSSION TITLE
how to reduce the base64 size returned by canvas.toDataURL()

### DISCUSSION THREAD

**Discussion by chj-damon** [iddan/react-native-canvas#259] [Q&A]
I use react-native-canvas to transform an image to base64, the original size is about 32kb, but after the base64 is generated, the size increased to 580kb, which is unbelievable huge.

I'm wondering if there is any solution to reduce the size.

the canvas API of web is different from react-native-canvas.

**Comment by ghost** [comment]
@chj-damon , I am sorry for the late reply... 

The thing about data url is that it is base64 encoded. Therefore it will make a file that is bigger (but usually up to 25%).
